frontend/public/components/catalog/_catalog.scss

```diff
@@ -24,6 +24,13 @@ $co-modal-ignore-warning-icon-width: 30px;
   padding: 0 0 ($grid-gutter-width / 2);
 }
 
+.co-catalog__body {
+  min-width: 590px; // Minimum shrinkable width of .co-catalog child elements => 30 + ((220 + 15) + (30 + 250 + 30)) + 15
```

Comment:
Values should be flipped so they're mobile first?

Reply:
switched